c.html

```diff
@@ -62,12 +62,12 @@
 	var validPassport = 0
 
 	// Regex
-	var regex4digits = new RegExp("([0-9]{4})$")
-	var regexHeight = new RegExp("([0-9][A-Za-z]{2})$")
-	var regexHexaColor = new RegExp("^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$")
-	var regexLeadingZeros = new RegExp(/[0-9]{4,100}$/);
+	var regex4digits = new RegExp("^([0-9]{4})$")
+	var regexHeight = new RegExp("^([0-9]+[A-Za-z]{2})$")
+	var regexHexaColor = new RegExp("^#([A-Fa-f0-9]{6})$")
```

Comment:
The specifications does not allow for a 3 letter hex color (even if this would be a valid CSS color)